src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -213,6 +213,38 @@ public override string ToString()
             }
         }
 
+        internal class StaticMirrorSigComparer : IEqualityComparer<StaticMirror>
```

Comment:
even though this is not public I would greatly appreciate /// summary explanation of this class.

Reply:
done